### PR TITLE
Oadp 1919 remove datamover snapshot hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
@anarnold97

GH# : 
OCPBUGS# : [OADP-1919](https://issues.redhat.com//browse/OADP-1919)
OSDOCS# : [OADP-1919](https://issues.redhat.com//browse/OADP-1919)

Added a procedure at the end to remove old snapshots from the DataMover bucket.

Version(s): 1.1.x and newer

Issue: [OADP-1919](https://issues.redhat.com//browse/OADP-1919) OADP-1.1: Data Mover can restore from incorrect snapshot when >1 VSR for the restore name and PVC name exists.

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
@akarol & @weshayutin - please see the first draft for [OADP-1919](https://issues.redhat.com/browse/OADP-1919). Thanks

**Comment by HagayVider1** [comment]
@weshayutin @akarol 
OADP-1919
See new section from line 87 to the end.
Please review

Thanks

**Comment by PrasadJoshi12** [comment]
@HagayVider1 can you please give a link to doc preview ? Thanks !

File: modules/oadp-backup-restore-cr-issues.adoc [hunk 1]
```diff
@@ -75,13 +75,78 @@ time="2023-02-17T16:33:13Z" level=error msg="Error backing up item" backup=opens
...
-. Create a new `Backup` CR.
\ No newline at end of file
+. Create a new `Backup` CR.
+
+[id="datamover-backup-cleanup_{context}"]
+== Datamover backup cleanup
+
+[NOTE]
+====
+Applies to OADP 1.1.x and newer.
```
Comment (PrasadJoshi12) [inline modules/oadp-backup-restore-cr-issues.adoc:94]: Applies to only 1.1.x.

File: modules/oadp-backup-restore-cr-issues.adoc [hunk 2]
```diff
@@ -75,13 +75,78 @@ time="2023-02-17T16:33:13Z" level=error msg="Error backing up item" backup=opens
...
+. Datamover completes
+
+volumeSnapshotBackup/VolumeSnapshotRestore CRs are still in the application namespace.
+
+* Datamover backup:
+
++
+[source, text]
+----
+oc delete vsb -n <app-ns> --all
```
Comment (PrasadJoshi12) [inline modules/oadp-backup-restore-cr-issues.adoc:116]: We are suggesting to cleanup all the things.

oc delete volumesnapshot -A --all
oc delete volumesnapshotcontent --all
oc delete pvc -n <protected-ns> --all
oc delete replicationsource -n <protected-ns> --all

File: modules/oadp-backup-restore-cr-issues.adoc (hunk 1 above)
Comment (weshayutin) [reply modules/oadp-backup-restore-cr-issues.adoc:94]: s/Applies to OADP 1.1.x and newer/Applies to OADP 1.1.x/g

**Comment by tareqalayan** [comment]
looks similar content like https://github.com/openshift/openshift-docs/pull/59894/files

File: modules/oadp-backup-restore-cr-issues.adoc (hunk 1 above)
Comment (HagayVider1) [reply modules/oadp-backup-restore-cr-issues.adoc:94]: Remover ".x and newer" from line 94

File: modules/oadp-backup-restore-cr-issues.adoc (hunk 2 above)
Comment (HagayVider1) [reply modules/oadp-backup-restore-cr-issues.adoc:116]: Added in lines 117-120

**Comment by HagayVider1** [comment]
> looks similar content like https://github.com/openshift/openshift-docs/pull/59894/files

@tareqalayan 
Should I also change "backing-up-applications.adoc"?

**Review by weshayutin — APPROVED** [review]

**Comment by anarnold97** [comment]
@HagayVider1 - please can you check with Richard about the content of this PR? If he has already covered and provided the content required, then close this PR

Thanks

**Comment by anarnold97** [comment]
@HagayVider1  - please close this PR
Thanks

**Pull request closed without merging**